src/jit/block.h

```diff
@@ -925,14 +925,12 @@ typedef unsigned weight_t;             // Type used to hold block and edge weigh
 
     bool endsWithJmpMethod(Compiler *comp);
 
-#if FEATURE_FASTTAILCALL
```

Comment:
I had a close look at the definition of these methods and they seems to be correct for targets where FEATURE_FASTTAILCALL is zero.  You can ignore above comment of mine.